app/api/support/DownstreamResponseMappingSupport.scala

```diff
@@ -32,6 +32,10 @@ trait DownstreamResponseMappingSupport {
         logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - No matching stub was found")
         RuleIncorrectGovTestScenarioError
       }
+      case "INVALID_CORRELATION_ID" | "INVALID_CORRELATIONID" => {
+        logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - An internal server error occurred")
```

Comment:
Not sure what we log elsewhere but we probably don't want to hide the reason for the error as it will make it trickier to investigate and fix if it happens. It's not that an internal error has occurred: we've received a very specific error. 
Maybe:
```suggestion
        logger.warn(s"[${logContext.controllerName}] [${logContext.endpointName}] - An invalid correlation id error occurred")
```